app/templates/components/location-select.hbs

```diff
@@ -1,8 +1,10 @@
-{{select-list
-  value=locationPicker.selectedLocation
-  content=locationPicker.subLocationList
-  class="form-control"
-  optionValuePath="selectObject"
-  optionLabelPath="selectObject.locationNameWithQuantity"
-  action=locationChange
-}}
\ No newline at end of file
+{{#em-form model=this submitButton=false as |form|}}
```

Comment:
Now I'm wondering if this doesn't introduce the same issue "form within a form" since we use that in a component which could possibly be used in a form. Could you please doublecheck?

Reply:
And sorry for the PR ping-pong

Reply:
Yeah, it is used in a form. This one still works in Firefox unlike the first one, but it is still incorrect. Also quantity-conv.hbs does this, maybe others. Thanks for catching this, I just didn't know that about the HTML form tag. I'll have to think about how to handle this.

Reply:
Maybe there is already a rapid form which contains a component. In that case you should be able to pass the rapid form reference in the component and you're done.

Reply:
Ok, will try that.

Reply:
Some of these are multiple levels of component down from the {{#em-form}}, quantity-conv.hbs for example is inventory/edit/template -> (partial 'inv-purchase') -> quantity-calc -> quantity-conv, so it could end up a little confusing. Of course the alternative is another native HTML. It's up to your preference.

Reply:
OK, it works but I had to also add "model=this" in the individual {{form.select}}'s.